<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>159</width>
    <height>205</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>41</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(213, 255, 255);
	background-color: rgb(208, 234, 236);
color: rgb(0, 0, 0);
border: 1px solid dark;
}
QPushButton:pressed {
background-color: rgb(183, 206, 207);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>41</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(213, 255, 255);
	background-color: rgb(208, 234, 236);
color: rgb(0, 0, 0);
border: 1px solid dark;
}
QPushButton:pressed {
background-color: rgb(183, 206, 207);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>41</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 
	background-color:   rgb(244, 118, 15);
  color: white; 
  border: 1px solid white;
}

QPushButton:pressed {

	background-color: rgb(163, 76, 9);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>41</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(213, 255, 255);
	background-color: rgb(208, 234, 236);
color: rgb(0, 0, 0);
border: 1px solid dark;
}
QPushButton:pressed {
background-color: rgb(183, 206, 207);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>81</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(213, 255, 255);
	background-color: rgb(208, 234, 236);
color: rgb(0, 0, 0);
border: 1px solid dark;
}
QPushButton:pressed {
background-color: rgb(183, 206, 207);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>81</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(213, 255, 255);
	background-color: rgb(208, 234, 236);
color: rgb(0, 0, 0);
border: 1px solid dark;
}
QPushButton:pressed {
background-color: rgb(183, 206, 207);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>81</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(213, 255, 255);
	background-color: rgb(208, 234, 236);
color: rgb(0, 0, 0);
border: 1px solid dark;
}
QPushButton:pressed {
background-color: rgb(183, 206, 207);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>81</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 
	background-color:   rgb(244, 118, 15);
  color: white; 
  border: 1px solid white;
}

QPushButton:pressed {

	background-color: rgb(163, 76, 9);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>121</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(213, 255, 255);
	background-color: rgb(208, 234, 236);
color: rgb(0, 0, 0);
border: 1px solid dark;
}
QPushButton:pressed {
background-color: rgb(183, 206, 207);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>121</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(213, 255, 255);
	background-color: rgb(208, 234, 236);
color: rgb(0, 0, 0);
border: 1px solid dark;
}
QPushButton:pressed {
background-color: rgb(183, 206, 207);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>121</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(213, 255, 255);
	background-color: rgb(208, 234, 236);
color: rgb(0, 0, 0);
border: 1px solid dark;
}
QPushButton:pressed {
background-color: rgb(183, 206, 207);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>121</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 
	background-color:   rgb(244, 118, 15);
  color: white; 
  border: 1px solid white;
}

QPushButton:pressed {

	background-color: rgb(163, 76, 9);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>161</y>
      <width>42</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(182, 49, 49);
	color: rgb(255, 255, 255);
border: solid dark;
}
QPushButton:pressed {
 
	background-color: rgb(136, 67, 67);
}
</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>161</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(213, 255, 255);
	background-color: rgb(208, 234, 236);
color: rgb(0, 0, 0);
border: 1px solid dark;
}
QPushButton:pressed {
background-color: rgb(183, 206, 207);
}</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>161</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(213, 255, 255);
	background-color: rgb(208, 234, 236);
color: rgb(0, 0, 0);
border: 1px solid dark;
}
QPushButton:pressed {
background-color: rgb(183, 206, 207);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eq">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>161</y>
      <width>40</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 
	background-color:   rgb(244, 118, 15);
  color: white; 
  border: 1px solid white;
}

QPushButton:pressed {

	background-color: rgb(163, 76, 9);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>1</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color: rgb(255, 255, 255);
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}

background-color : gray;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>1</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 
	background-color:   rgb(244, 118, 15);
  color: white; 
  border: 1px solid white;
}

QPushButton:pressed {

	background-color: rgb(163, 76, 9);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <zorder>status</zorder>
   <zorder>pushButton_AC</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_div</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_plus</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_minus</zorder>
   <zorder>pushButton_dot</zorder>
   <zorder>pushButton_0</zorder>
   <zorder>pushButton_eq</zorder>
   <zorder>pushButton_x</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
